pkg/sql/catalog/catprivilege/default_privilege.go

```diff
@@ -358,16 +359,19 @@ func GetUserPrivilegesForObject(
 	// If ForAllRoles is specified, we can return early.
 	// ForAllRoles is not a real role and does not have implicit default privileges
 	// for itself.
-	if !p.IsExplicitRole() {
-		return userPrivileges
-	}
-	userProto := p.GetExplicitRole().UserProto
-	if GetRoleHasAllPrivilegesOnTargetObject(&p, targetObject) {
-		return append(userPrivileges, catpb.UserPrivileges{
-			UserProto:  userProto,
-			Privileges: privilege.ALL.Mask(),
-		})
-	}
+	//if !p.IsExplicitRole() {
```

Comment:
Are we planning to delete this block?

Reply:
yes, this part should have just been deleted. the thing i wasn't sure about with the TODO is how this logic differs from what's on line 338